.github/workflows/collector.yml

```
@@ -28,13 +33,26 @@ env:
   ADDRESS_SANITIZER: ${{ contains(github.event.pull_request.labels.*.name, 'address-sanitizer') }}
 
 jobs:
-  build-collector-image:
-    name: Build Collector
+  prepare-build-collector:
+    name: Prepare builders for collector
     runs-on: ubuntu-24.04
+    outputs:
+      local-exclude: ${{ steps.arch.outputs.local-exclude || '' }}
```

Looks like an empty string is not valid JSON according to GHA:
```
Error when evaluating 'strategy' for job 'build-collector-image'. stackrox/collector/.github/workflows/collector.yml@c69bf5ada5bd5cc96efe18f4d0a082f9c726433c (Line: 54, Col: 18): Error parsing fromJson,stackrox/collector/.github/workflows/collector.yml@c69bf5ada5bd5cc96efe18f4d0a082f9c726433c (Line: 54, Col: 18): Error reading JToken from JsonReader. Path '', line 0, position 0.,stackrox/collector/.github/workflows/collector.yml@c69bf5ada5bd5cc96efe18f4d0a082f9c726433c (Line: 54, Col: 18): Unexpected value ''
```

https://github.com/stackrox/collector/actions/runs/14743136370?pr=2106

```suggestion
      local-exclude: ${{ steps.arch.outputs.local-exclude || '[]' }}
```